### DISCUSSION TITLE
How to use Kafka Cluster in flask-socketio

### DISCUSSION THREAD

**Discussion by boomjuice** [miguelgrinberg/Flask-SocketIO#1850] [Q&A] [answered]
I have a kafka cluster config like this .
`message_queue: kafka://10.0.42.149:9092,10.0.42.154:9092,10.0.42.156:9092`

so I pass the config to socketio, it raises error because the bootstrap_servers is invalid.
<img width="1114" alt="image" src="https://user-images.githubusercontent.com/40669311/180153971-a170714c-f663-4089-a2aa-7148f2ba553e.png">

I get one solution that passing 'client_manager' by myself

Is there have any other solution?

**Comment by boomjuice** [comment, marked as answer]
<img width="797" alt="image" src="https://user-images.githubusercontent.com/40669311/180174808-1ac2a07c-ca12-4a27-89a7-8fe7beaff5b5.png">
🧐 I find a method to solve it but i am not sure it makes sense

**Reply by miguelgrinberg** [reply]
Flask-SocketIO does not currently support a cluster of URLs, so my recommendation is that you create your client manager instance. The comma-separated IP address is not a standard URL format, so while you are free to use that format in your own application I will not be implementing something like that here.
